### ISSUE TITLE
Clarity on range normalization of inputs

### ISSUE THREAD

**Issue by hari-aindra** [google-research/big_transfer#16]
Normalization range for input to the network is not clearly defined. 

 The tutorial says it should be normalized to the range 0-1 [here](https://github.com/google-research/big_transfer/blob/147d22da1975030d09abd4a94a0534013636f019/colabs/big_transfer_tf2.ipynb) 
`  features['image'] = tf.cast(features['image'], tf.float32) / 255.0`

The training code on the other hand normalizes it to -1 to +1 [here](https://github.com/google-research/big_transfer/blob/5a518d91e818f05dd868d47b932121540b41dcbe/input_pipeline_tf2_or_jax.py#L107)
`    im = (im - 127.5) / 127.5`

Among these, which is the correct one?

**Comment by hari-aindra** [comment]
That solves the issue. Thanks

**Comment by hari-aindra** [comment]
Do the tensor-hub model with 0-1 scaling have same performance(Accuracy) as of the original BIT models ?

**Comment by lucasb-eyer** [comment]
Yes, they are exactly the same models and just do the conversion internally.
